clients/imodels-client-management/src/base/internal/AxiosRestClient.ts

```diff
@@ -92,5 +96,28 @@ export class AxiosRestClient implements RestClient {
       throw error;
     }
   }
+
+  private async executeWithRetry<TBody>(requestFunc: () => Promise<AxiosResponse<TBody>>): Promise<AxiosResponse<TBody>> {
+    let retriesInvoked = 0;
+    for (;;) {
+      try {
+        return await requestFunc();
+      } catch (error: unknown) {
+        if (
+          this._retryPolicy === null ||
+          retriesInvoked >= this._retryPolicy.maxRetries ||
+          retriesInvoked >= 10 ||
```

Comment:
nitpick: move the value into a constant in this class with a descriptive name, like `retryCountUpperLimit`

Reply:
updated